[2024-10-26T21:46:50.647+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-26T21:46:50.750+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: coinmarket_extraction.fetch_coin_data scheduled__2024-10-26T05:40:00+00:00 [queued]>
[2024-10-26T21:46:50.799+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: coinmarket_extraction.fetch_coin_data scheduled__2024-10-26T05:40:00+00:00 [queued]>
[2024-10-26T21:46:50.801+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-26T21:46:50.837+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): fetch_coin_data> on 2024-10-26 05:40:00+00:00
[2024-10-26T21:46:50.856+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'coinmarket_extraction', 'fetch_coin_data', 'scheduled__2024-10-26T05:40:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/extractor.py', '--cfg-path', '/tmp/tmpgzeok86f']
[2024-10-26T21:46:50.848+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1265) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-26T21:46:50.867+0000] {standard_task_runner.py:72} INFO - Started process 1292 to run task
[2024-10-26T21:46:50.866+0000] {standard_task_runner.py:105} INFO - Job 50: Subtask fetch_coin_data
[2024-10-26T21:46:51.053+0000] {task_command.py:467} INFO - Running <TaskInstance: coinmarket_extraction.fetch_coin_data scheduled__2024-10-26T05:40:00+00:00 [running]> on host c94222e110db
[2024-10-26T21:46:51.392+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='coinmarket_extraction' AIRFLOW_CTX_TASK_ID='fetch_coin_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-26T05:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-26T05:40:00+00:00'
[2024-10-26T21:46:51.398+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-26T21:46:51.399+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-26T21:46:51.400+0000] {logging_mixin.py:190} INFO - Current task name:fetch_coin_data state:running start_date:2024-10-26 21:46:50.757639+00:00
[2024-10-26T21:46:51.402+0000] {logging_mixin.py:190} INFO - Dag name:coinmarket_extraction and current dag run status:running
[2024-10-26T21:46:51.406+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-26T21:46:51.615+0000] {python.py:240} INFO - Done. Returned value was: Falha na requisição: 429
[2024-10-26T21:46:51.696+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-26T21:46:51.698+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=coinmarket_extraction, task_id=fetch_coin_data, run_id=scheduled__2024-10-26T05:40:00+00:00, execution_date=20241026T054000, start_date=20241026T214650, end_date=20241026T214651
[2024-10-26T21:46:51.718+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-26T21:46:51.719+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-26T21:46:51.725+0000] {logging_mixin.py:190} INFO - Dag name:coinmarket_extraction queued_at:2024-10-26 21:46:47.093123+00:00
[2024-10-26T21:46:51.726+0000] {logging_mixin.py:190} INFO - Task hostname:c94222e110db operator:PythonOperator
[2024-10-26T21:46:51.770+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-26T21:46:51.879+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-26T21:46:51.895+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
